.github/workflows/ci.yml

```diff
@@ -20,6 +20,14 @@ jobs:
       - run: poetry install
       - name: Run CLI tests
         run: poetry run make test
+      - name: Set path for candle and light
+        env:
+          WIX_PATH: ${{ env.wix }}
+        run: echo "$env:WIX_PATH\bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+      - name: Build .exe
+        run: poetry run python .\setup-windows.py build
+      - name: Build the MSI installer
+        run: poetry run .\install\windows\build-app.bat
```

Comment:
I believe this step already calls `python .\setup-windows.py build` but it probably won't run successfully because it expects some signing stuff. I had to comment out some things to make it work locally.

Reply:
Oh. Never mind. I see it's a WIP. Testing CI stuff is complicated.